satellite/contact/contact_test.go

```diff
@@ -51,3 +51,20 @@ func TestSatelliteContactEndpoint(t *testing.T) {
 		require.Equal(t, ident.PeerIdentity(), peerID)
 	})
 }
+
+func TestFetchInfo(t *testing.T) {
+	testplanet.Run(t, testplanet.Config{
+		SatelliteCount: 1, StorageNodeCount: 1, UplinkCount: 0,
+	}, func(t *testing.T, ctx *testcontext.Context, planet *testplanet.Planet) {
+		nodeDossier := planet.StorageNodes[0].Local()
+		node := pb.Node{Id: nodeDossier.Id, Address: nodeDossier.Address}
+
+		resp, err := planet.Satellites[0].Contact.Service.FetchInfo(ctx, node)
+		require.NotNil(t, resp)
+		require.NoError(t, err)
+		require.Equal(t, nodeDossier.Type, resp.Type)
+		require.Equal(t, &nodeDossier.Operator, resp.Operator)
+		require.Equal(t, &nodeDossier.Capacity, resp.Capacity)
+		require.Equal(t, nodeDossier.Version.Version, resp.Version.Version)
```

Comment:
I'm not sure why the whole Version struct didn't match... I think the timestamp was different

Reply:
timezones np